incubator/command-config/src/test/resources/io/aklivity/zilla/runtime/command/config/internal/openapi/http/proxy/complete/zilla.yaml

```diff
@@ -0,0 +1,124 @@
+name: example
+bindings:
+  tcp_server0:
+    type: tcp
+    kind: server
+    options:
+      host: 0.0.0.0
+      port:
+      - 8080
+      - 9090
+    routes:
+    - exit: http_server0
+      when:
+      - port: 8080
+    - exit: tls_server0
+      when:
+      - port: 9090
+  tls_server0:
+    vault: server
+    type: tls
+    kind: server
+    options:
+      keys:
+      - "${{env.TLS_SERVER_KEYS}}"
+      sni:
+      - "${{env.TLS_SERVER_SNI}}"
+      alpn:
+      - "${{env.TLS_SERVER_ALPN}}"
+    exit: http_server0
+  http_server0:
+    type: http
+    kind: server
+    options:
+      access-control:
+        policy: cross-origin
+      authorization:
+        jwt0:
+          credentials:
+            headers:
+              authorization: "Bearer {credentials}"
+    routes:
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items
+          :method: GET
+      guarded:
+        jwt0: []
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items
+          :method: POST
+      guarded:
+        jwt0: []
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: GET
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: PUT
+    - exit: http_client0
+      when:
+      - headers:
+          :path: /items/*
+          :method: DELETE
+  http_client0:
+    type: http
+    kind: client
+    exit: tls_client0
+  tls_client0:
+    vault: client
+    type: tls
+    kind: client
+    options:
+      trust:
+      - "${{env.TLS_CLIENT_TRUST}}"
+      trustcacerts: true
+      sni:
+      - "${{env.TLS_CLIENT_SNI}}"
+      alpn:
+      - "${{env.TLS_CLIENT_ALPN}}"
+    exit: tcp_client0
+  tcp_client0:
+    type: tcp
+    kind: client
+    options:
+      host: "${{env.TCP_CLIENT_HOST}}"
+      port: "${{env.TCP_CLIENT_PORT}}"
```

Comment:
Let's remove the quotes here, should be an integer.

Reply:
Done, pls chk. We need to do another round of string transformation after the yaml output has been generated to remove the unnecessary quotes. If you have an idea for a more efficient approach pls LMK.